### ISSUE TITLE
Table auto adjust

### ISSUE THREAD

**Issue by Skemp45** [RaspberryPints/RaspberryPints#29]
When we moved to a table we lost the ability for our taplist to auto adjust. look at screen shot to see what I mean. before I could stretch it across both my screen and it would adjust to the size but now it stops after so far. 
![liststreach](https://f.cloud.github.com/assets/6318869/1894817/0bf0c39a-7aff-11e3-948b-971095debdcb.png)

**Comment by thadius856** [comment]
It's still auto-filling, but only as far as the single longest line of text in the tasting notes, I think.

Look at the date/time. It's alllllll the way to the right. So the table itself is still stretching.

**Comment by thadius856** [comment]
Not able to test at the moment (weighing hops, boil about to start), but this looks like the problem.

```
table{
        border-collapse:collapse;
}

table th,
table td{
        border-left: 1px white dashed;
        padding:5px 10px;
        text-align: center;
        color: white;
}

table th:first-child,
table td:first-child{
        border-left: none;
}

table thead{
        height:60px;
        background-color: rgba(153,153,153,.3);
        padding-top: 3px;
}

table thead th{
        height:55px;
        font: .65em arial, verdana, sans-serif;
        color: white;
        font-weight: bold;
        font-style:italic;
        vertical-align: top;
}

table .srm,
table .tap-num{
        width: 8%;
}

table .ibu{
        width: 14%;
}

table .name{
        width: 44%;
        text-align: left;
}
table .abv{
        width: 14%;
}
table .keg{
        width: 15%;
}
```

Shouldn't the first declaration `table {}` get `width: 100%;`?

Seems like it would be easy to assign the column widths in px instead of %, then set `table .name` to `width: auto;`.

**Comment by JayUnt** [comment]
Good catch. Table should get 100%. As for the other columns having a pixel
eighth, we could do that if you guys want. I did percent so the columns set
width columns expand a but with a stretched screen. But I can def see
pixels being the better way to go. I don't think you need to set the .name.
column's width. I this it's automatically auto. I can make these changes.

**Comment by JayUnt** [comment]
just committed the fix.  my laptop didnt allow it to stretch far enough so i didnt see the issue.  but i jsut tested by zooming out, and it looks to work for me now.  

I also changed all of the columns, except name, to use a pixel width, so that column will stretch to its heart's content.

**Comment by thadius856** [comment]
You commit isn't appearing on the master yet.

**Comment by JayUnt** [comment]
actaully the commit failed.  "skipping obstructing working copy'  have to look into this and will commit.

**Comment by JayUnt** [comment]
ok we are good to go.

**Comment by thadius856** [comment]
Yep, tested fine here. Fill wells at both 1080x1920 and 1200x1920. Closing issue.

Btw, I got TortoiseGit working. TortoiseSvn was causing me headaches. I'm now editing from a folder on my desktop named 'var www' which I have WinSCP set to keep synced with the Pi's /var/www. Takes about 3-5 seconds once I save in Notepad++ before they're replicated on the server, but otherwise not bad at all.

**Comment by JayUnt** [comment]
Ill have to check out tortoisegit.   sounds good.

On Sat, Jan 11, 2014 at 6:00 PM, thadius856 notifications@github.comwrote:

> Closed #29 https://github.com/thadius856/Digital-Taplist/issues/29.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/thadius856/Digital-Taplist/issues/29
> .

## 

Jason Unterman
